hknweb/events/views.py

```diff
@@ -151,6 +152,14 @@ class EventUpdateView(UpdateView):
     form_class = EventUpdateForm
     template_name_suffix = '_edit'
 
+    def get_initial(self):
```

Comment:
Where is this function used?

Reply:
I believe get_initial is something that Django uses to populate the form with pre-existing data before it is actually rendered on the template

Reply:
Yeah so it's used in the parent class